docs/tutorial_step6.md

```diff
@@ -41,7 +45,7 @@ Then, once the activity will be open, he will share the activity by clicking on
 
 ![](images/tutorial_step6_5.png)
 
-From your Neighborhood view you will see the icon of the activity suddenly appear near the Michaël's icon. Just pass the mouse on the activity icon and click the Join menu and you will be able to join Michaël.
+When you go to Neighborhood view(click on **Neighborhood view** icon from homepage as shown previously) from your(LIONEL's) browser, you will see the icon of the activity suddenly appear near the Michaël's icon. Just pass the mouse on the activity icon and click the Join menu and you will be able to join Michaël.
```

Comment:
It seems clear in the current paragraph.

Reply:
In tutorial `Neighborhood view` is referred to both of following icon
* `Neighborhood view` button from homepage
* `Neighborhood view` button from any particular activity page

This make so confusing that tutorial is suggesting to join `Neighborhood view` from activity page from one user client and to open `Neighborhood view` from homepage from another user client. So I though adding some more clarification would be helpful.